### DISCUSSION TITLE
Termostato Tuya con Valvole radiatori Wifi

### DISCUSSION THREAD

**Discussion by lbottoni** [indomus/forum#1705] [GENERALE - Discussioni generiche]
Salve,
vorrei sostuire il mio termostato di casa a 2 fili con uno smart compatibile Tuya (per compatibilità di tutta la domotica di casa) e fino a qui nessun problema. Quello che poi vorrei fare e poter gestire ogni singola stanza con una valvola termostatica nei radiatori delle stanze. Quello che non mi torna è se io metto 21° nel termostato a muro , 25° il radiatore della sala, 18° il radiatore della cucina e in casa il termostato a muro rileva 21.5° , mi viene da pensare che il termostato a muro è quello che ha l'ultima parola e spenerebbe il contatto con la caldaia, corretto? qualcuno ha già esperienza reale ?

vi allego 2 immagini per far capire cosa intendo per termostato a mura tuya smart e valvola tuya smart

**TERMOSTATO MURO**
![termostato](https://user-images.githubusercontent.com/13313/170455829-fa6a9282-8324-40be-82fc-426d68e7b8d5.png)
**VALVOLA RADIATORE**
![valvola termosifone](https://user-images.githubusercontent.com/13313/170455835-e974c75e-01ea-4db6-8905-ad7bd3dcfa4f.png)

**Comment by maurominella** [comment]
Non è diverso da come ha sempre funzionato anche quando non c'era la domotica: il termostato a muro comanda l'accensione caldaia, mentre le valvole controllano l'apertura dei termosifoni. Quindi se la caldaia è spenta è vero che la valvola del calorifero è aperta, ma l'acqua calda non viene prodotta quindi risulta freddo anche se in realtà è "acceso".
Ho avuto le stesse valvole (poi cambiate perché queste non mi sono piaciute), la soluzione che ho sempre adottato è tenere il termostato al massimo e lavorare solo sulle valvole. L'unico possibile inconveniente è che certe caldaie vanno in blocco se sono accese mentre tutte le valvole sono chiuse, io l'ho risolto con un puffer e, prima di mettere il puffer, tenendo una valvola sempre aperta.
La tecnologia comunque consentirebbe di mandare al termostato il comando di spegnimento al termostato quando tutte le valvole salgono sopra la temperatura impostata (e viceversa quando la prima scende sotto temperatura), bisogna capire quanto ci si vuole impegnare per farlo.

**Reply by lbottoni** [reply]
rapida occhiata su "puffer" non sapendo cosa sia ma mi pare un'oggetto molto costoso e non vale la candela. piuttosto mi converebbe mettere un relè smart al posto del termostato e lasciare la chiusura e apertura alle valvole, eliminado completamente il termostato.
starno che non esistano soluzioni pratiche a questa piccolezza

**Reply by maurominella** [reply]
mi sono meravigliato anche io ma purtroppo sembra che non sia un problema sentito. Pure più serio per me è il problema della temperatura rilevata dalle valvole, che è falsata dal fatto che siano posizionate proprio in fianco alla sorgente di calore. Non è una differenza di qualche decimo, per me parliamo anche di 3-4 gradi che rende impossibile un controllo realistico. Quindi ho acquistato i termometri zigbee per ogni stanzsa ma la procedura è comunque complessa da implementare e da mantenere

**Reply by lbottoni** [reply]
Grazie mille per la condivisione di esperienza

**Reply by 50494554524F** [reply]
io uso dei termometri "xiaomi thermometer 2" in ogni stanza (moddati con firmware pvvx), costano/costavano 3-3,50€ a seconda del periodo di offerte... puoi comprare anche dei magneti da 2cm di diametro e ci attacchi la patch fornita per attaccarla dove vuoi; togli il coperchio dietro il termometro e la batteria farà da placca magnetica. naturalmente mettere un termometro in ogni stanza funzionerà solo se lascerai le porte chiuse, per ovvi motivi...

**Comment by maurominella** [comment]
Però il problema non è sapere io qual è la temperatura giusta, il problema è fare in modo che le valvole si basino su questi termometri esterni invece che sul loro termometro interno, per decidere quando aprirsi a chiudersi.

**Reply by 50494554524F** [reply]
quello sta alla tua fantasia... nel mio caso, per esempio, uso uno switch group, quando un termometro del gruppo segna la temperatura giusta il termostato generico apre la valvola lo switch va su on ed una automazione manda il comando al pulsante per accendere la caldaia... puoi adattarlo al tuo caso

**Reply by lbottoni** [reply]
Ok, il tuo ragionamento mi torna su quello che riflettevo, ma mi daresti indicazioni su cosa intendi per valvole generiche controllate da uno switch? Hai modelli per farmi capire e hai l'automatismo di esempio x caso?
Lo switch sarebbe un relè lo so😉

**Reply by maurominella** [reply]
> quello sta alla tua fantasia... nel mio caso, per esempio, uso uno switch group, quando un termometro del gruppo segna la temperatura giusta il termostato generico apre la valvola lo switch va su on ed una automazione manda il comando al pulsante per accendere la caldaia... puoi adattarlo al tuo caso

io intendo: come fai ad aprire la valvola (non accendere la caldaia) usando il termometro da 3 euro esterno alla valvola, considerato che la valvola ha già un suo termometro interno che la apre e la chiude a seconda della temperatura che rileva? Anche se tu mandassi il segnale di chiusura ma il termometro interno della valvola rileva temperatura sotto quella impostata, la riapre subito. E viceversa se provi ad aprirla quando il suo termometro interno è sopra la temperatura impostatata: la valvola si richiude subito. Ho provato anche a forzare la temperatura "rilevata" dal termometro interno alla valvola sovrascrivendola con quella del termometro esterno, ma anche in quel caso viene riportata alla temperatura della termovalvola (diversa da quella del termometro) dopo pochi secondi. Puoi provare ad agire sul valore di tolleranza della valvola, ma a quel punto non puoi più regolare la temperatura della stanza usando i comandi sulla valvola, che è la cosa normale che ci si aspetterebbe da una valvola con regolatore, no? Va bene che siamo tecnologici, ma non ha senso costringere una persona ad usare l'app davanti al termosifone. Ovviamente se prendi una soluzione MAx! questo lo fa perché ti danno un termometro esterno già accoppiato, ma costa un occhio della testa (80 euro a termometro, valvola esclusa, gateway escluso, tutto blindato e poi quando cambiano linea sei costretto a ricomprare tutto!). Se qualcuno ha un'idea migliore di queste mi piacerebbe capire che cosa non ho pensato. Grazie

**Reply by 50494554524F** [reply]
> Ok, il tuo ragionamento mi torna su quello che riflettevo, ma mi daresti indicazioni su cosa intendi per valvole generiche controllate da uno switch? Hai modelli per farmi capire e hai l'automatismo di esempio x caso? Lo switch sarebbe un relè lo so😉

è il generic_thermostat di homeassistant

> io intendo: come fai ad aprire la valvola (non accendere la caldaia) usando il termometro da 3 euro esterno alla valvola, [CUT]

(se non puoi disabilitare il termometro interno della valvola) devi mettere la temperatura target al massimo possibile in modo che non intervenga, per fare tutto userai homeassistant...
